tracpro/contacts/models.py

```diff
@@ -89,6 +89,10 @@ class Contact(models.Model):
         'groups.Group', null=True, verbose_name=_("Reporter group"),
         related_name='contacts',
         help_text=_("Reporter group to which this contact belongs."))
+    groups = models.ManyToManyField(
```

Comment:
If you want to change the groups on the contact update page, the field is necessary. Otherwise, I think it isn´t necessary.